techmatters/flex-plugins
# Gian chi 1732 followup

## Description
This PR addresses the following QA note:
> Another question: in the empty state when no counsellors are available, how easy or difficult would it be to display text like “No counsellors are available right now”? Sorry this wasn’t explicitly stated in this ticket’s acceptance criteria. It’s a nice to have, not a release blocker.

Now, if there's no counselor available to transfer, the `Agents` tabs will be replaced as shown in the imaage.
![Screenshot from 2023-04-07 17-46-58](https://user-images.githubusercontent.com/15805319/230637932-efbc6dde-8a77-4571-a49e-f105ad6891fe.png)


### Checklist
- [x] [Corresponding issue has been opened](https://tech-matters.atlassian.net/browse/CHI-1732)
- [n/a] Feature flags added
- [n/a] Tested for chat contacts
- [n/a] Tested for call contacts

### Verification steps
- Make sure you are the only "available" counselor and try to transfer a task; you should see as in the image above.
- Repeat but with at least one other counselor "available"; the filtered list of "available" counselors should display.
